engine/hatchery/local/local.go

```diff
@@ -33,6 +33,14 @@ func (h *HatcheryLocal) ApplyConfiguration(cfg interface{}) error {
 		return fmt.Errorf("Invalid configuration")
 	}
 
+	// h.Client = cdsclient.NewService(h.Config.API.HTTP.URL, 60*time.Second)
```

Comment:
Comment are normal, I avoid update to many hatchery for now, waiting branch from @bnjjj 
They will be useful to have hatchery as a CDS Services